include/intx/int128.hpp

```diff
@@ -366,25 +380,6 @@ constexpr uint128 constexpr_umul(uint64_t x, uint64_t y) noexcept
     return {hi, lo};
 }
 
-/// Full unsigned multiplication 64 x 64 -> 128.
-inline uint128 umul(uint64_t x, uint64_t y) noexcept
-{
-#if defined(__SIZEOF_INT128__)
-    #pragma GCC diagnostic push
-    #pragma GCC diagnostic ignored "-Wpedantic"
-    using builtin_uint128 = unsigned __int128;
-    const auto p = builtin_uint128{x} * y;
-    return {uint64_t(p >> 64), uint64_t(p)};
-    #pragma GCC diagnostic pop
-#elif defined(_MSC_VER)
```

Comment:
Why is this MSVC version removed?

Reply:
I think I decided it is not critical to have it fast on msvc.